docs/concepts/models/custom-model-settings.md

```diff
@@ -0,0 +1,229 @@
+- Connect to self-hosted or custom model endpoints
+- Create multiple variants of the same model with different settings
+
+## Creating and Using Custom Settings
+
+### Custom Models with Default Providers
+
+Create custom model configurations that use the default providers (no need to define providers yourself):
+
+```python
+from data_designer.essentials import (
+    CategorySamplerParams,
+    ChatCompletionInferenceParams,
+    DataDesigner,
+    DataDesignerConfigBuilder,
+    LLMTextColumnConfig,
+    ModelConfig,
+    SamplerColumnConfig,
+    SamplerType,
+)
+
+# Create custom models using default providers
+custom_models = [
+    # High-temperature creative model
+    ModelConfig(
+        alias="creative-writer",
+        model="nvidia/nvidia-nemotron-nano-9b-v2",
+        provider="nvidia",  # Uses default NVIDIA provider
+        inference_parameters=ChatCompletionInferenceParams(
+            temperature=1.2,  # Very creative
+            top_p=0.98,
+            max_tokens=4096,
+        ),
+    ),
+    # Low-temperature factual model
+    ModelConfig(
+        alias="fact-checker",
+        model="nvidia/nvidia-nemotron-nano-9b-v2",
+        provider="nvidia",  # Uses default NVIDIA provider
+        inference_parameters=ChatCompletionInferenceParams(
+            temperature=0.1,  # Very deterministic
+            top_p=0.9,
+            max_tokens=2048,
+        ),
+    ),
+]
+
+# Create DataDesigner (uses default providers)
+data_designer = DataDesigner()
+
+# Pass custom models to config builder
+config_builder = DataDesignerConfigBuilder(model_configs=custom_models)
+
+# Add a topic column using a categorical sampler
+config_builder.add_column(
+    SamplerColumnConfig(
+        name="topic",
+        sampler_type=SamplerType.CATEGORY,
+        params=CategorySamplerParams(
+            values=["Artificial Intelligence", "Space Exploration", "Ancient History", "Climate Science"],
+        ),
+    )
+)
+
+# Use your custom models
+config_builder.add_column(
+    LLMTextColumnConfig(
+        name="creative_story",
+        model_alias="creative-writer",
+        prompt="Write a creative short story about {{topic}}.",
+    )
+)
+
+config_builder.add_column(
+    LLMTextColumnConfig(
+        name="facts",
+        model_alias="fact-checker",
+        prompt="List 3 facts about {{topic}}.",
+    )
+)
+
+# Preview your dataset
+preview_result = data_designer.preview(config_builder=config_builder)
+preview_result.display_sample_record()
+```
+
+!!! note "Default Providers Always Available"
+    When you only specify `model_configs`, the default model providers (NVIDIA and OpenAI) are still available. You only need to create custom providers if you want to connect to different endpoints or modify provider settings.
+
+!!! tip "Mixing Custom and Default Models"
+    When you provide custom `model_configs` to `DataDesignerConfigBuilder`, they **replace** the defaults entirely. To add custom models while keeping the defaults:
+
```

Comment:
Oh dang, I hadn't considered this. Might be helpful to explicitly say what is happening. Something like: "To use custom model configs _in addition_ to the default configs, use the `add_model_config` method:"